### PR TITLE
Some suggestions

### PR DESCRIPTION
**Pull request by deathaxe**
I use FuzzyFileNav as alternative to the ordinary Open File command and like it very much.

I found some things I'd suggest to add/change.

1. How about adding the main commands to the command pallet?
2. The normal context menu should contain text related commands only. Therefore I'd suggest to move the "Nav ..." items from the context menu to the tabs context menu.
3. I bound the "Fuzzy Here..." command to <kbd>ctrl+shift+o</kbd> a long time ago to quickly access it. I wonder whether it should be the default key binding? Not sure about it.
4. Some of the files in the repository don't need to be part of a sublime-package and should therefore be `export-ignored` to reduce package size and keep it clean.

### CODE REVIEW

File: Default (Linux).sublime-keymap [hunk 1]
```diff
@@ -1,5 +1,10 @@
 [
     {
+        "keys": ["ctrl+shift+o"],
```
Comment (facelessuser) [inline Default (Linux).sublime-keymap]: I'm generally against default keymaps that aren't limited by context.  I personally bind FuzzyFileNav as you have done here, but I don't force that on anyone.

File: Main.sublime-menu [hunk 2]
```diff
@@ -1,5 +1,34 @@
 [
     {
+        "id": "file",
```
Comment (facelessuser) [inline Main.sublime-menu]: I'm probably less interested in adding menu items (as I rarely use these), but I'll think about it.

File: .gitattributes [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## docs sources
+doc_theme/ export-ignore
+docs/ export-ignore
+mkdocs.yml export-ignore
+
+## unit testing
+tests/ export-ignore
+run_tests.sh export-ignore
+
```
Comment (facelessuser) [inline .gitattributes]: I am unaware of how Package Control does its packaging.  So this actually excludes these from the package?

Also, including documents isn't always a bad thing as people can look at it locally if they like.

File: Default.sublime-commands [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        "args": {"paths": []}
+    },
+    {
+        "caption": "Fuzzy Project Folders",
+        "command": "fuzzy_project_folder_load"
+    },
+    {
+        "caption": "Fuzzy BookMarks",
+        "command": "fuzzy_bookmarks_load"
+    }
```
Comment (facelessuser) [inline Default.sublime-commands]: These are probably useful.  So these I may pull  in.

File: .gitattributes (hunk 3 above)
Comment (deathaxe) [reply .gitattributes]: Package Control runs an `git archive` command via GitHub API. All files in the `.gitattributes` file which are marked with `export-ignore` are therefore not part of a sublime-package.

Leaving docs untouched might not be a bad idea. As they are part of a zip I wonder who finds them there?

File: Default (Linux).sublime-keymap (hunk 1 above)
Comment (deathaxe) [reply Default (Linux).sublime-keymap]: I can follow your arguments. Key bindings may not be too important if the commands are found in the command pallet.

Comment (facelessuser) [reply Default (Linux).sublime-keymap]: Yeah, people can bind whatever they want.

File: .gitattributes (hunk 3 above)
Comment (facelessuser) [reply .gitattributes]: I don't know, but I generally view shipping them with the product makes sense. Whether anyone finds or looks at them or not isn't a concern for me.

File: Main.sublime-menu (hunk 2 above)
Comment (facelessuser) [reply Main.sublime-menu]: If someone specifically requests them, maybe.  But I haven't really received such a request.

**Comment by facelessuser** [comment]
So final verdict:

1. No to keybindings and menus items.  Menu items *might* get added in the future if I get some requests for it, but generally Sublime is built around the palette.  Though some of my plugins do have some menu items, they are usually infrequently accessed things that don't make sense cluttering up the palette.  Though packages like BracketHighlighter do have commands there, that is only because i was new to package development when that was added, and I haven't removed it only for the 1% who are used to finding those commands there.
2. I think I'm okay with "Context" becoming "Tab Context" menu as the options don't actually operate on the view content.
3. I think I'm okay with the export changes as long as we don't exclude document source.  I like shipping the documents so people can access them if they are offline (even if it is bundled in a zip). One could argue that if I am going to include documents, I should provide a built version (which I could do), but meh.

So these changes would have to be made for this to get pulled.

**Comment by deathaxe** [comment]
Do you prefer fixup! commits or ordinary ones including the changes?

**Comment by deathaxe** [comment]
OK, rebased and removed unwanted commits for a clean history.

File: .gitattributes (hunk 3 above)
Comment (facelessuser) [reply .gitattributes]: Are we sure this stuff doesn't break Travis? I'm getting an error about a missing `.travis.yml` file: https://travis-ci.org/facelessuser/FuzzyFileNav/builds/296883556?utm_source=github_status&utm_medium=notification. I'm trying to restart the build to see what happens.

Comment (facelessuser) [reply .gitattributes]: I suspect maybe the way this pull was updated broke Travis's ability to run.  I may just have to do this on a separate branch to verify changes.

**Comment by facelessuser** [comment]
Changes have been merged in the https://github.com/facelessuser/FuzzyFileNav/tree/dev/2.0 branch.  Some development is going on over there, but once it is finished, these changes will come back to the main.

**Pull request closed without merging**